scalahost/sbt/src/sbt-test/sbt-scalahost/semantic-example/test

```diff
@@ -0,0 +1,11 @@
+> app/testOnly Fat
+> set every scalametaSemanticdb := ScalametaSemanticdb.Slim
+> clean
+> app/testOnly Slim
+> clean
+> set every scalametaSemanticdb := ScalametaSemanticdb.Fat
```

Comment:
`every` works here, but you need to be careful because it overrides all the keys in every scope. `set scalametaSemanticdb in Compile := ScalametaSemanticdb.Fat` and its counterpart for `Test` should work. I would prefer the last ones since they are more precise.
  
See `help set`:
```
set [every] <setting-expression>

	Applies the given setting to the current project:
	  1) Constructs the expression provided as an argument by compiling and loading it.
	  2) Appends the new setting to the current project's settings.
	  3) Re-evaluates the build's settings.

	This command does not rebuild the build definitions, plugins, or configurations.
	It does not automatically persist the setting(s) either.
	To persist the setting(s), run 'session save' or 'session save-all'.

	If 'every' is specified, the setting is evaluated in the current context
	and the resulting value is used in every scope.  This overrides the value
	bound to the key everywhere.

```